include/rumble.h

```diff
@@ -0,0 +1,54 @@
+#ifndef RUMBLE_H
+#define RUMBLE_H
+
+#include "ultra64.h"
+
+#define RUMBLE_MAX_REQUESTS 64
+
+typedef enum {
+    RUMBLE_STATE_CLEAR,
+    RUMBLE_STATE_RUNNING,
+    RUMBLE_STATE_RESET
+} RumbleState;
+
+typedef struct {
+    /* 0x000 */ u8 rumbleEnable[MAXCONTROLLERS];
+
+    /* 0x004 */ u8 reqStrengths[RUMBLE_MAX_REQUESTS];       // Source strength modulated by distance to the source
+    /* 0x044 */ u8 reqDurations[RUMBLE_MAX_REQUESTS];       // Duration until decreaseRate kicks in
+    /* 0x084 */ u8 reqDecreaseRates[RUMBLE_MAX_REQUESTS];   // Decreases the strength by this much every Vertical Retrace, once the strength hits 0 the request slot is freed
+    /* 0x0C4 */ u8 reqAccumulator[RUMBLE_MAX_REQUESTS];     // Starts at 0, incremented by the strength every Vertical Retrace
+
+    /* 0x104 */ u8 state;
+    /* 0x105 */ u8 updateEnabled;
+    /* 0x106 */ u16 onTimer;    // Duration for which there has been an active rumble request running
+    /* 0x108 */ u16 offTimer;   // Duration for which there has not been an active rumble request running, capped at 5
+
+    /* 0x10A */ u8 overrideStrength;    // overrides requests with these parameters
+    /* 0x10B */ u8 overrideDuration;
+    /* 0x10C */ u8 overrideDecreaseRate;
+    /* 0x10D */ u8 overrideAccumulator;
```

Comment:
small thing but imo no need for the line breaks in the struct

Reply:
fwiw I like them but I'm okay with not having them